.github/workflows/ci.yml

```diff
@@ -1,29 +1,38 @@
 name: neptune-fastai
 on: [push]
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          pip install -e .[dev]
+      - name: Lint
+        run: |
+          python -m pylint -j 0 -f parseable neptune_fastai tests
   test:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # Windows builds fail because of the file header check - not clear what causes this.
-        # There are no pylint-file-header issues about this, and Windows builds succeed in the old domain
-        os: [ubuntu-latest, macos-latest] #, windows-latest]
```

Comment:
Why have you removed this comment?

Reply:
It was only reasonable for linting (which was moved to additional job that runs only on ubuntu)

Reply:
But don't we want to test on windows?

Reply:
Disabled as it failures on Windows due to lack of some additional steps (dataset download)